### PR TITLE
feat(UI-1003): redesign intro page

### PR DESCRIPTION
**Pull request by J1za**
## Description
Design - the design is not final yet, because we still don't have the whole content, but let's start by developing the new elements in it:

https://www.figma.com/design/klCZL68fUd6pw0DEu0Sj7f/Autokitteh%3A-Look-%26-Feel-design-versions?node-id=3728-129049&t=uRi2LIHPfwLokW91-4
## Linear Ticket
https://linear.app/autokitteh/issue/UI-1003/welcome-page-re-design
## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/templates/appLayout.tsx [hunk 1]
```diff
@@ -48,16 +49,16 @@ export const AppLayout = ({ className, hideTopbar }: { className?: string; hideT
 
 	return (
 		<div className={appLayoutClasses}>
-			<Sidebar />
+			{!isTable ? <Sidebar /> : null}
```
Comment (RonenMars) [inline src/components/templates/appLayout.tsx:52]: if we don't load and display `<Sidebar />`, would the projectList load?

Comment (J1za) [reply src/components/templates/appLayout.tsx:52]: if we don't load and display `<Sidebar />`, projectList don't load

Comment (RonenMars) [reply src/components/templates/appLayout.tsx:52]: On mobile, we would like to show the menu (the sidebar).
Not on the intro/welcome page, but on any other page we will need it.

File: src/hooks/useWindowDimensions.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+
+const getWindowDimensions = () => {
+	const { innerHeight: height, innerWidth: width } = window;
+
+	const isDesktop = width >= 1680;
+	const isDesktopSmall = width <= 1440;
+	const isLaptop = width <= 1199;
+	const isLaptopSmall = width <= 1024;
+	const isTable = width <= 768;
```
[the remark this answers is not in the document]
Comment (J1za) [reply src/hooks/useWindowDimensions.ts]: renamed

**Review by RonenMars — APPROVED** [review]

**Pull request merged**